### PR TITLE
pulsarctl 3.3.0.3 (new formula)

### PR DESCRIPTION
**Pull request by branchv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

---

Needed a livecheck due to an old [calver tag](https://github.com/streamnative/pulsarctl/tree/v20200607-rc) and `github_latest` isn't accurate either ([latest](https://github.com/streamnative/pulsarctl/releases/latest) is `v3.0.5.4` but their [tap](https://github.com/streamnative/homebrew-streamnative/blob/master/Formula/pulsarctl.rb) is on `v3.3.0.1`)

### CODE REVIEW

**Comment by chenrui333** [comment]
why not use latest v3.0.5.4 ?

**Comment by branchvincent** [comment]
Cause
> [latest](https://github.com/streamnative/pulsarctl/releases/latest) is v3.0.5.4 but their [tap](https://github.com/streamnative/homebrew-streamnative/blob/master/Formula/pulsarctl.rb) is on v3.3.0.1

So it seems like they are maintaining multiple minor versions in parallel

**Comment by chenrui333** [comment]
I feel like it is more of goreleaser flow issue :), can you raise an upstream issue to confirm about this? I'd prefer 3.0.5.4

File: Formula/p/pulsarctl.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class Pulsarctl < Formula
+  desc "CLI for Apache Pulsar written in Go"
+  homepage "https://streamnative.io/"
+  url "https://github.com/streamnative/pulsarctl/archive/refs/tags/v3.3.0.3.tar.gz"
+  sha256 "8171726d64266b569fae1c6b7fadcc99f90e4117aef2e71ff54f992afbc7939e"
+  license "Apache-2.0"
+  head "https://github.com/streamnative/pulsarctl.git", branch: "master"
+
+  livecheck do
+    url :head
```
Comment (carlocab) [inline Formula/p/pulsarctl.rb:10]:
```suggestion
    url :stable
```

**Comment by branchvincent** [comment]
Are they using goreleaser? I don't see a `.goreleaser.yaml`

But https://github.com/streamnative/pulsarctl/pull/980#issuecomment-1418740172 confirms they are maintaining muliple minor versions in parallel, as they track the same minor versions as pulsar (and the latest there is [3.3.x](https://github.com/apache/pulsar/releases/latest))

**Review by carlocab — APPROVED** [review]

**Comment by chenrui333** [comment]
yeah, they dont use it, it would be better that they use it to automate the tap updates.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**